src/main/java/org/sysc4907/votingsystem/VotingSystemApplication.java

```diff
@@ -2,11 +2,47 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
 
 @SpringBootApplication
 public class VotingSystemApplication {
 	public static void main(String[] args) {
+		// Create accounts_db if it does not already exist
+		createDB("accounts_db", "postgres", "postgres");
```

Comment:
username and password cannot be hard coded. get these values from application.properties file.